src/com/facebook/buck/jvm/core/HasJavaAbi.java

```diff
@@ -43,18 +43,18 @@
   Flavor SOURCE_ONLY_ABI_FLAVOR = InternalFlavor.of("source-only-abi");
   Flavor VERIFIED_SOURCE_ABI_FLAVOR = InternalFlavor.of("verified-source-abi");
 
-  static BuildTarget getClassAbiJar(BuildTarget libraryTarget) {
```

Comment:
Any changes in this file? Why do you need to move this method?

Reply:
just moved the method to be in accordance with others, but will revert back